model/src/test/kotlin/utils/FileArchiverTest.kt

```diff
@@ -66,6 +68,13 @@ class FileArchiverTest : StringSpec() {
         file.writeText(path)
     }
 
+    private fun createEmptyFile(path: String): File {
+        val file = workingDir.resolve(path)
+        file.parentFile.safeMkdirs()
+        file.createNewFile()
+        return file
+    }
+
```

Comment:
Instead of introducing this, you could generalize the existing function to

```kotlin
private fun createFile(path: String, write: File.() -> Unit = { writeText(path) }) {
    val file = workingDir.resolve(path)
    file.parentFile.safeMkdirs()
    file.write()
}
```

and then in your new test use

    createFile("License") { writeBytes(byteArrayOf(0xFF.toByte(), 0xD8.toByte())) }

and

    createFile("License") { writeText("ぁあぃいぅうぇえぉおかが") }

respectively.